CHANGES.md

```diff
@@ -7,7 +7,7 @@
 - #1188 Feature request: drag and drop file to HTTP Upload
 - #1268 Switch from SASS variables to CSS custom properties
 - #1278 Replace the default avatar with a SVG version
-
+- #1033 show send buuton:true issue fixes
```

Comment:
Please fix the typo...

Reply:
@jcbrand . Sorry but what is typo. is there any spelling mistake

Reply:
"typo" means typing mistake.

"buuton" should be "button".

Please change the text to: `#1033 Setting show_send_button to true didn't work`